### PR TITLE
Feature/update project model

### PR DESCRIPTION
**Pull request by gorold**
### Context 
- Updated Project model
- Minor changes to superuser creation

### Changes 
- Updated Project model to include:
  - cover (ImageField)
  - description (TextField)
  - company (ForeignKey)
  - owners (ManyToMany)
- Project CRUD API permissions
  - List provides list of projects only which the user is in "owners"
  - If the user is_superuser, list returns all projects
  - Similarly for RetrieveUpdateDestroy
- Deleted new_admin.py, replaced with manage.py createadmin 

### Considerations
REQUIRED_FIELDS in Project model enforces company and invitation uuid in createsuperuser, but is required by Djoser to ensure those fields are filled in the API. Thus, we extend createsuperuser and make a manage.py createadmin function.

### CODE REVIEW

File: secretsauce/apps/account/urls.py [hunk 1]
```diff
@@ -18,4 +18,6 @@
     path('', include('djoser.urls.authtoken')),
     path('invitation/', views.InvitationCreator.as_view()),
     path('invitation/<uuid:pk>', views.InvitationDetail.as_view()),
+    path('company/', views.CompanyList.as_view()),
+    path('company/', views.CompanyDetail.as_view()),
```
Comment (eugenechia95) [inline secretsauce/apps/account/urls.py:22]: wrong path name? Cant access invidividual companies

File: secretsauce/apps/portal/serializers.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class ProjectSerializer(serializers.ModelSerializer):
```
Comment (eugenechia95) [inline secretsauce/apps/portal/serializers.py:13]: Just noticed this. What is line 12 for? Not relevant right?

**Comment by gorold** [comment]
> Everything else looks good. Nice work with the createadmin 💯
> I only tested the projects endpoints but I suppose you tested the rest.
> 
> For the manytomanyfield to update project users, it still doesnt work right? You will be fixing in a separate PR?

This one is working already, just send a Patch request, with an owners array. It will override the current owners. If you want to send like an add owner or delete owner, then it should be in another view I think.

File: secretsauce/apps/portal/serializers.py (hunk 2 above)
Comment (gorold) [reply secretsauce/apps/portal/serializers.py]: Yup, not relevant, removed.

File: secretsauce/apps/account/urls.py (hunk 1 above)
Comment (gorold) [reply secretsauce/apps/account/urls.py]: Changed in latest commit

**Pull request merged**